src/schema/rules/sidecars/asl.yaml

```diff
@@ -31,7 +31,24 @@ MRIASLTextOnlySliceTiming:
     - suffix == "asl"
     - sidecar.MRAcquisitionType == "2D"
     fields:
-        SliceTiming: REQUIRED
+        SliceTiming:
+            level: REQUIRED
+            code:
+                name: SLICE_TIMING_NOT_DEFINED_2D_ASL
+                message: |
+                    "You should define SliceTiming', in case 'SequenceType' is set to a
+                    2D sequence. 'SliceTiming' is the time at which each slice was
+                    acquired within each volume (frame) of the acquisition. Slice timing
+                    is not slice order -- rather, it is a list of times containing the
+                    time (in seconds) of each slice acquisition in relation to the beginning
+                    of volume acquisition. The list goes through the slices along the slice
+                    axis in the slice encoding dimension (see below). Note that to ensure the
+                    proper interpretation of the `SliceTiming` field, it is important to check
+                    if the OPTIONAL `SliceEncodingDirection` exists. In particular, if
+                    `SliceEncodingDirection` is negative, the entries in `SliceTiming` are
+                    defined in reverse order with respect to the slice axis, such that the
+                    final entry in the `SliceTiming` list is the time of acquisition of slice 0.
+                    Without this parameter slice time correction will not be possible."
```

Comment:
```suggestion
            code: SLICE_TIMING_NOT_DEFINED_2D_ASL
            message: |
                You should define SliceTiming', in case 'SequenceType' is set to a
                2D sequence. 'SliceTiming' is the time at which each slice was
                acquired within each volume (frame) of the acquisition. Slice timing
                is not slice order -- rather, it is a list of times containing the
                time (in seconds) of each slice acquisition in relation to the beginning
                of volume acquisition. The list goes through the slices along the slice
                axis in the slice encoding dimension (see below). Note that to ensure the
                proper interpretation of the `SliceTiming` field, it is important to check
                if the OPTIONAL `SliceEncodingDirection` exists. In particular, if
                `SliceEncodingDirection` is negative, the entries in `SliceTiming` are
                defined in reverse order with respect to the slice axis, such that the
                final entry in the `SliceTiming` list is the time of acquisition of slice 0.
                Without this parameter slice time correction will not be possible.
```

Reply:
NO!